openmdao/drivers/scipy_optimizer.py

```diff
@@ -141,6 +142,17 @@ def __init__(self):
 
         self.cite = CITATIONS
 
+    def _get_name(self):
+        """
+        Get name of current driver.
```

Comment:
I see this is the same docstring as in pyOptSparseDriver, but it seems incorrect... it returns the name of the cuurent optimizer, not the name of the driver (which is `ScipyOptimizeDriver`)

Reply:
fixed